pkg/web_app/lib/src/search.dart

```diff
@@ -5,12 +5,34 @@
 import 'dart:html';
 
 void setupSearch() {
+  _setEventForKeyboardShortcut();
   _setEventForSearchInput();
   _setEventForFiltersToggle();
   _setEventForSortControl();
   _setEventForCheckboxChanges();
 }
 
+void _setEventForKeyboardShortcut() {
+  final inputElem = document.querySelector('input.site-header-search-input');
+  if (inputElem != null && inputElem is InputElement) {
+    window.onKeyPress.listen((e) {
+      // Ignore keys other than the shortcut key:
+      if (e.key != '/') return;
+
+      // Only trigger the input field and steal focus when nothing is focused,
+      // or when the focused element is not an input element.
+      final active = document.activeElement;
+      final isRestricted = active is InputElement || active is TextAreaElement;
+      if (!isRestricted) {
+        inputElem.focus();
```

Comment:
Maybe: send an event to analytics when shortcut is triggered

Reply:
Done. PTAL.